lib/Receiver.js

```diff
@@ -80,22 +60,14 @@ class Receiver {
 
   add(data) {
     if (this.dead) return;
-    const dataLength = data.length;
-    if (dataLength == 0) return;
-    if (this.expectBuffer == null) {
-      this.overflow.push(data);
-      return;
-    }
-    const toRead = Math.min(dataLength, this.expectBuffer.length - this.expectOffset);
-    fastCopy(toRead, data, this.expectBuffer, this.expectOffset);
-    this.expectOffset += toRead;
-    if (toRead < dataLength) {
-      this.overflow.push(data.slice(toRead));
-    }
-    while (this.expectBuffer && this.expectOffset == this.expectBuffer.length) {
-      const bufferForHandler = this.expectBuffer;
-      this.expectBuffer = null;
-      this.expectOffset = 0;
+    if (!this.expectBytes) return;
```

Comment:
I think it happens if the receiver gets data after being reset(). The only time I saw this was in the 'will not crash when it receives an unhandled opcode' test.